manifests/update.pp

```diff
@@ -88,10 +88,19 @@
   validate_bool($reboot_required)
 
   # Windows update service.
-  service { $service:
-    ensure => 'running',
-    enable => true,
-  }
+  case $::kernelmajversion {
+    '6.3', '10.0' : {
```

Comment:
I think it is better to remove ensure => running at all. The wsuaserv is always started on-demand.

This is how the puppetlabs-wsus_client is handling it. See the MODULES-2420's 10th comment. 

Let me know your thoughts and I'll commit the change.